src/collection.ts

```diff
@@ -338,4 +350,33 @@ export class Collection<T extends TigrisCollectionType> {
 			});
 		});
 	}
+
+	events(
+		events: EventsCallback<T>,
+		tx ?: Session,
```

Comment:
subscription of events takes transaction optionally as input?

Reply:
removed